INTELLIJ_SETUP.md

```diff
@@ -0,0 +1,59 @@
+# IntelliJ Setup
+This document contains some examples and instructions on how to get IntelliJ setup to run local debuging and test setups of Druid.
+
+## Code Style
+The Code Style is available in XML format at [druid_intellij_formatting.xml](druid_intellij_formatting.xml) and can be [imported into IntelliJ](https://www.jetbrains.com/help/idea/2017.1/copying-code-style-settings.html).
+
+## MySQL
```

Comment:
I don't have a means of testing that :) if you have better info to add here, that would be cool but probably outside of this PR.